exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,39 @@
+function groupPercentage(num1, num2){
+    let totalInGroup = num1 + num2;    
+    return calculatePercentage = Math.round((num1 / totalInGroup) * 100);
+}
+
+function message(numberOfStudents, numberOfMentors){
+    let percentageOfStudents = groupPercentage(numberOfStudents, numberOfMentors);
+    let percentageOfMentors = groupPercentage (numberOfMentors, numberOfStudents);
+    return displayPercentage = `Percentage students: ${percentageOfStudents}%\nPercentage students: ${percentageOfMentors}%`;
```

Comment:
The displayPercentage variable assignment is redundant. You don't need to include this. Just return the string.